src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/RectangleSelection.js

```diff
@@ -88,13 +88,12 @@ class RawRectangleSelectionComponent extends React.Component<Props> {
                 minWidth,
                 minHeight
             ),
-            new PositionNormalizer(
-                containerWidth,
-                containerHeight
-            ),
         ];
 
         if (minWidth && minHeight) {
+            // It's important that the RatioNormalizer is added before the PositionNormalizer, because it may alter the
```

Comment:
It looks like we already changed this in the past (#5399), but the changes got lost in some other PR 🙈 
Hopefully the comment will prevent this from happening again 🙂